src/services/types.ts

```diff
@@ -242,7 +250,7 @@ namespace ts {
         getEncodedSyntacticClassifications(fileName: string, span: TextSpan): Classifications;
         getEncodedSemanticClassifications(fileName: string, span: TextSpan): Classifications;
 
-        getCompletionsAtPosition(fileName: string, position: number, options: GetCompletionsAtPositionOptions | undefined): CompletionInfo;
+        getCompletionsAtPosition(fileName: string, position: number, options: GetCompletionsAtPositionOptions | undefined, settings: ServicesSettings | undefined): CompletionInfo;

```

Comment:
make it optional instead so that the api doesn't break

Reply:
Like with the `opoptions: GetCompletionsAtPositionOptions | undefined,`, this is a types breaking change but will handle it gracefully at runtime -- we want to be reminding people to send us the settings though.